script/Arcueid/Day 10/10_00_ARC10_8_2.txt

```diff
@@ -47,7 +47,7 @@ The savage emotions I felt are gone,
 [sha:882a5b5232422929107f77ce52686c919819de75]{
 -- Page 868, Offset 26779.
 -- 「―――なんて―――こと、を」
-"―――How―――could, I..."
+"―――How――could, I..."
```

Comment:
I dunno if the dashes are appropriate at all, tbh.
The first one at least seems weird.